stl_text/ops/tokenizers/whitespace_tokenizer.py

```diff
@@ -1,44 +1,46 @@
+import torch
 import torch.nn as nn
 from typing import Dict, List, Optional
+from collections import OrderedDict
 from iopath.common.file_io import PathManager
+from torchtext.experimental.vocab import (
+    build_vocab_from_text_file,
+    Vocab,
+)
+from torchtext._torchtext import Vocab as VocabPybind 
+import torchtext.experimental.transforms as transforms
 
 
-class WhitespaceTokenizer(nn.Module):
-    def __init__(self, vocab_path: Optional[str] = None, trainable=False, speed: int = 0):
+class WhitespaceTokenizer(nn.Module): 
+    def __init__(self, vocab_path: Optional[str] = None, trainable: bool = False):
         super(WhitespaceTokenizer, self).__init__()
         self.trainable = trainable
-        self.speed = speed  # mock a real tokenizer: slowing down tokenization speed
+        self.tokenizer = transforms.basic_english_normalize().to_ivalue()
```

Comment:
Do we need `basic_english_normalize` here? Can we keep the original tokenizer and switch to torchtext vocab?

Reply:
Agreed!